content/id/guides/directory-structure/layouts.md

```diff
@@ -0,0 +1,173 @@
+  - question: Anda dapat dengan mudahnya mengubah nama direktori layouts tanpa konfigurasi khusus.
+    answers:
+      - benar
+      - salah
+    correctAnswer: salah
+  - question: Apa nama halaman tata letak default?
+    answers:
+      - layout.vue
+      - default.vue
+      - defaultLayout.vue
+    correctAnswer: default.vue
+  - question: Komponen apa saja yang harus ada pada seluruh layout?
+    answers:
+      - <Nuxt />
+      - <NuxtLink />
+      - <RouterView />
+    correctAnswer: <Nuxt />
+  - question: Anda dapat menambah komponen lain ke dalam layout anda
+    answers:
+      - benar
+      - salah
+    correctAnswer: benar
+  - question: Untuk menambah sebuah layout kustom, anda perlu membuat sebuah file `.vue` dan ke folder manakah file itu akan ditempatkan?
+    answers:
+      - layout
+      - layouts
+      - page
+    correctAnswer: layouts
+  - question: Bagaimana caranya anda memberitahukan sebuah halaman untuk menerapkan blog layout?
+    answers:
+      - "layout: 'blog'"
+      - "name: 'blog'"
+      - 'blog: true'
+    correctAnswer: "layout: 'blog'"
+  - question: Pada direktori manakah halaman error seharusnya ditambahkan?
+    answers:
+      - pages
+      - layouts
+      - errors
+    correctAnswer: layouts
+  - question: Anda perlu menambahkan komponen `<Nuxt>` pada halaman error?
+    answers:
+      - benar
+      - salah
+    correctAnswer: salah
+  - question: Anda dapat menggunakan custom layout untuk halaman error
+    answers:
+      - benar
+      - salah
+    correctAnswer: benar
+  - question: Halaman error ditampilkan ketika terjadi kesalahan pada saat rendering di sisi server (server side rendering)?
+    answers:
+      - benar
+      - salah
+    correctAnswer: salah
+---
+
+Layouts adalah bagian yang sangatlah membantu ketika anda ingin mengubah tampilan dan nuansa pada aplikasi Nuxt.js anda, baik memasukkan _sidebar_ maupun memiliki layout berbeda untuk perangkat mobile dan desktop.
+
+<base-alert>
+
+_Direktori ini tidak dapat diubah namanya tanpa konfigurasi khusus._
+
+</base-alert>
+
+## Default Layout
+
+Anda dapat memperluas file layout utama dengan menambahkan file `layouts/default.vue`. Layout tersebut akan diterapkan pada seluruh halaman yang tidak memiliki layout khusus. Pastikan penggunaan komponen `<Nuxt>` ketika pembuatan layout untuk benar-benar menyertakan page component.
+
+Semua yang anda butuhkan pada layout adalah tiga baris kode berikut yang akan me-render page component.
+
+```html{}[layouts/default.vue]
+<template>
+  <Nuxt />
+</template>
+```
+
+Anda dapat menambahkan komponen-komponen lainnya seperti Navigation, Header, Footer, dan lain-lain.
+
+```html{}[layouts/default.vue]
+<template>
+  <TheHeader />
+  <Nuxt />
+  <TheFooter />
+</template>
+```
+
+<base-alert type="info">
+
+Jika anda memiliki ["components set to true"](/guides/directory-structure/components), maka tidak perlu lagi menambahkan pernyataan-pernyataan import pada komponen-komponen tersebut.
```

Comment:
`components set to true` kalau ngga diterjemahkan mungkin bisa dijadikan format miring